HW5/loop_in_a_loop.py

```diff
@@ -0,0 +1,12 @@
+#1
+#1 2
+#1 2 3
+#1 2 3 4
+#1 2 3 4 5
+
+for x in range(1,7):
```

Comment:
nit: add space after comma
```suggestion
for x in range(1, 7):
```